src/shogun/classifier/LDA.cpp

```diff
@@ -66,8 +68,7 @@ bool CLDA::train_machine(CFeatures *data)
 	REQUIRE(m_labels, "Labels for the given features are not specified!\n")
 	REQUIRE(
 	    m_labels->get_label_type() == LT_BINARY,
-	    "The labels should of type"
-	    " CBinaryLabels! you provided %s \n",
+	    "The labels should of type CBinaryLabels! you provided %s \n",
```

Comment:
General question: Is this multiclass LDA? If so, why does it assert binary labels?
I think there should be no distinction (class wise) between binary and multiclass LDA